docker-compose.yml

```
@@ -72,7 +80,7 @@ services:
     command:
       - "--providers.docker.exposedByDefault=false"
       - "--providers.docker.network=cvat"
-      - "--entryPoints.web.address=:8080"
+      - "--entryPoints.web.address=:80"
```

Leave this as it was, and instead use a different port mapping below
```suggestion
      - "--entryPoints.web.address=:8080"
```